toolkit/imageconfigs/packagelists/developer-packages.json

```diff
@@ -15,6 +15,11 @@
         "curl-devel",
         "cmake",
         "flex",
-        "ninja-build"
+        "ninja-build",
```

Comment:
Small ask: since we're already here, could we put these in alphabetical order? I think we tried to do that up until `wget` and something happened.:)